mountainsort/src/processors/fit_stage.cpp

```diff
@@ -114,7 +113,8 @@ bool fit_stage(const QString& timeseries_path, const QString& firings_path, cons
             if (to_use[i] == 1) {
                 something_changed = true;
                 num_added++;
-                subtract_scaled_template(M * T, X.dataPtr(0, times_to_try[i] - Tmid), templates.dataPtr(0, 0, labels_to_try[i] - 1));
+                long tt = (long)(times_to_try[i] - Tmid + 0.5);
```

Comment:
I didn't realize that. However I like showing the cast to long explicitly because it shows that we are rounding here. In future we are going to handle fractional timepoints throughout.